projects/canopy/src/lib/forms/radio/radio.notes.ts

```diff
@@ -33,7 +35,13 @@ and in your HTML:
 | \`\`id\`\` | HTML ID attribute, auto generated if not provided | string | 'lg-radio-group-id-\${nextUniqueId++}' | No |
 | \`\`name\`\` | Set the name value for all inputs in the group, auto-generated if not provided | string | 'lg-radio-group-\${nextUniqueId++}' | No |
 | \`\`value\`\` | Set the default checked radio button, must match the value of the radio button | string | null | No |
-| \`\`inline\`\` | If true, displays the radio buttons inline rather than stacked | boolean | false | No |
+${
+  name === 'Radio'
+    ? `| \`\`inline\`\` | If true, displays the radio buttons inline rather than stacked | boolean | false | No |`
+    : name === 'Segment'
+    ? `| \`\`stack\`\` | Stack the radio buttons at a given breakpoint | 'sm', 'md', 'lg' | undefined | No |`
```

Comment:
I think we have a 'false' option here as well, I was wondering if we should set the default to `false` rather than undefined?

Reply:
Updated.